src/emulate.c

```diff
@@ -28,6 +28,7 @@ static inline int isnanf(float x)
 extern struct target_ops gdbstub_ops;
 #endif
 
+#include "common.h"
```

Comment:
You don't have to include `common.h` because the build system already specifies.

Reply:
The reason why header `common.h` is not explicitly included in C source is that we have to ensure that it is always the first header to be included, otherwise we might suffer from potential macro conflicts.

Reply:
Since we rely on `clang-format` for indention, the order of header files might be changed, and programmers would not be aware of it.